watch-parties/src/main/java/com/google/sps/data/Chat.java

```diff
@@ -0,0 +1,72 @@
+package com.google.sps.data;
+import java.util.Date;
+
+public final class Chat {
+ 
+  private String message;
```

Comment:
Are message, authorId, and timestamp supposed to be mutable? It seems like these fields would not be rewritten after creation of the Chat object. Justify their mutability or make these final.